packages/callout/src/Callout/Callout.spec.tsx

```diff
@@ -27,15 +27,12 @@ describe('packages/callout', () => {
 
   for (const key of Object.keys(Variant)) {
     const variant = Variant[key as keyof typeof Variant];
-    const icon = headerIcons[variant];
     const label = headerLabels[variant];
 
     describe(`for variant ${variant}`, () => {
-      test(`renders icon "${icon.displayName}" and label "${label}" in header"`, () => {
+      test(`renders label "${label}" in header"`, () => {
```

Comment:
I think it was here before your change but there's an extra `"` at the end of this test statement